pkg/resource/util.go

```diff
@@ -48,6 +48,14 @@ func ReadPassword(c client.Client, namespace, secretName string) (string, error)
 }
 
 func GetOceanbaseOperationManagerFromOBCluster(c client.Client, logger *logr.Logger, obcluster *v1alpha1.OBCluster) (*operation.OceanbaseOperationManager, error) {
+	return getOperationClient(c, logger, obcluster, oceanbaseconst.OperatorUser, oceanbaseconst.SysTenant, obcluster.Spec.UserSecrets.Operator)
+}
+
+func GetTenantOperationClient(c client.Client, logger *logr.Logger, obcluster *v1alpha1.OBCluster, tenantName string) (*operation.OceanbaseOperationManager, error) {
+	return getOperationClient(c, logger, obcluster, oceanbaseconst.RootUser, tenantName, tenantName+"-credential")
```

Comment:
maybe make credential a parameter

Reply:
OK, it will make function parameters more clear